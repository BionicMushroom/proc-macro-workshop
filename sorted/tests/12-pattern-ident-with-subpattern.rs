#[sorted::check]
fn main() {
    let x = 10;

    #[sorted]
    match x {
        ref num @ 1 => println!("{num}"),
        0 => println!("0"),
        _ => println!("everything else"),
    }
}
