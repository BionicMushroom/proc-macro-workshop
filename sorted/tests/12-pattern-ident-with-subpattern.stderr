error: 0 should sort before `ref num @ 1`
 --> tests/12-pattern-ident-with-subpattern.rs:8:9
  |
8 |         0 => println!("0"),
  |         ^
